pynicotine/__init__.py

```diff
@@ -24,20 +24,20 @@ def check_arguments():
 
     import argparse
     from pynicotine.config import config
-    parser = argparse.ArgumentParser(description=_("Nicotine+ is a Soulseek client"), add_help=False)
 
-    # Visible arguments
-    parser.add_argument(
-        "-h", "--help", action="help",
-        help=_("show this help message and exit")
+    parser = argparse.ArgumentParser(
+        description="%s %s - %s" % (config.application_name, config.version, config.summary),
+        epilog="%s <%s>" % (config.author, config.website_url)
```

Comment:
Are there other programs that do this? URL might be fine, not sure if author is relevant here.

Reply:
> other programs that do this?

Some have informational help text or copyright information and such like:

```
$ geany --help
...

Report bugs to https://github.com/geany/geany/issues.

$ 
```

> URL might be fine, not sure if author is relevant here.

The URL looks odd on it's own, the `man nicotine` page is the inspiration.

Reply:
> The URL looks odd on it's own, the `man nicotine` page is the inspiration.

That section mentions the author of the manpage though. I think something like `Website: https://nicotine-plus.org` would make more sense here.

Reply:
Yes, I had thought about that it would have been my first choice, but unfortunately we don't currently have a translatable string for "Website", which was why I went for author instead. It can be changed but the translators won't have time for the release of 3.2.3.

I suggest we merge this now and then change it to "Website: %s" at a later date.

Reply:
I don't think this is an important change for 3.2.3. You can include a Website string here, and I'll just backport the description change to 3.2.3.